core/unit_test/CMakeLists.txt

```diff
@@ -1235,12 +1235,10 @@ if (NOT KOKKOS_HAS_TRILINOS)
       INPUT TestDeviceAndThreads.py
       ${USE_SOURCE_PERMISSIONS_WHEN_SUPPORTED}
     )
-    if(NOT Kokkos_ENABLE_OPENMPTARGET)  # FIXME_OPENMPTARGET does not select the right device
```

Comment:
Oversight in #5492

Reply:
Looks like OpenMPTarget still doesn't do the right thing.
@rgayatri23 please fix.
Withdrawing the commit in the meantime.